master/getting-started/kubernetes/installation/azure.md

```diff
@@ -15,5 +17,6 @@ If the out-of-the-box solutions listed above don't meet your requirements, you c
 on Azure using one of our [self-hosted manifests][self-hosted], or by [integrating {{site.prodname}} with your own configuration management][integration-guide].
 
 [acs-engine]: https://github.com/Azure/acs-engine/blob/master/docs/kubernetes.md
+[azure-reference]: {{site.baseurl}}/{{page.version}}/reference/public-cloud/azure
```

Comment:
I think you just need to make this a relative URL to avoid the htmlproofer error.

Reply:
Noop it does not help, i did `[azure-reference]: ../../../reference/public-cloud/azure`.

Problem in other part, generated `reference/public-cloud/azure.html` page has this 
```
<link rel="canonical" href="https://docs.projectcalico.org/latest/reference/public-cloud/azure" />
```

So as i understand it will be checking real upstream, which does not have this page.

Reply:
Fixed for now by adding `no_canonical: true` to the page. Please take look on content itself.